internal/backend/dashboardsvc/connections.go

```diff
@@ -12,10 +13,21 @@ import (
 )
 
 func (s Svc) initConnections() {
+	// These paths should correspond to the ones enriched in the connection service.
+
 	s.Muxes.Auth.HandleFunc("GET /connections", s.connections)
 	s.Muxes.Auth.HandleFunc("GET /connections/{cid}", s.connection)
 	s.Muxes.Auth.HandleFunc("GET /connections/{id}/init", s.initConnection)
+	s.Muxes.Auth.HandleFunc("GET /connections/{id}/postinit", s.postInitConnection)
 	s.Muxes.Auth.HandleFunc("DELETE /connections/{id}/vars", s.rmAllConnectionVars)
+
+	// GET for browser, POST for automated.
```

Comment:
WDYM post for automated ?

Reply:
GET when used by browser, say from redirect.
POST when used by `fetch` from js.

Reply:
removed GET and POST, unified